tests/slash-menu.spec.ts

```diff
@@ -90,11 +92,12 @@ test.describe('slash menu should show and hide correctly', () => {
     await assertRichTexts(page, ['/_']);
 
     // And pressing backspace immediately should reappear the slash menu
-    await page.keyboard.press('Backspace');
+    await pressBackspace(page);
+    await waitNextFrame(page);
```

Comment:
```suggestion
```

I believe that there is no need to wait

Reply:
Just a try forgetted to removed, I will remove it.